code/modules/clothing/under/xenos/unathi.dm

```diff
@@ -107,4 +107,26 @@
 		item_state = "[initial(item_state)]_r"
 		to_chat(usr, span("notice", "You roll up your [src]'s sleeves."))
 		rolled_sleeves = TRUE
-	update_clothing_icon()
\ No newline at end of file
+	update_clothing_icon()
+
+/obj/item/clothing/suit/unathi/mantle/wrapping
+	name = "unathi wrappings"
+	desc = "Stitched together clothing with bandages covering them, looks tailored for an unathi."
+	description_fluff = "This is a very traditional Sinta wear for Th'akh shamans, most modern Sinta do not wear these."
+	icon = 'icons/obj/custom_items/karnaikai_wrappings.dmi'
```

Comment:
If its not a custom item anymore, change the icon path aswell and remove the custom item, unless there is a difference to the one added here